src/services/exile_service.py

```diff
@@ -142,18 +143,59 @@ async def get_user_exiles(logging_embed: discord.Embed, user: discord.User) -> s
 
     if len(exile_list) == 0:
         return "No exiles found for user"
+    else:
+        result = f"Exiles found for <@{user.id}>:"
+        for exile in exile_list:
+            exile_id = exile.exile_id
+            exile_reason = exile.reason
+            exile_start_epoch = round(exile.start_timestamp.timestamp())
+            exile_end_epoch = (
+                round(exile.end_timestamp.timestamp()) if exile.exile_status else None
+            )
+            exile_start_date = f"<t:{exile_start_epoch}:F>"
+            exile_end_date = (
+                f"<t:{exile_end_epoch}:F>" if exile_end_epoch else "Indefinite"
+            )
+            exile_type = ExileStatus(exile.exile_status).name
+
+            result = (
+                result
+                + f"\n* ID: {exile_id} | START DATE: {exile_start_date} | END DATE: {exile_end_date} | TYPE: {exile_type} | REASON: {exile_reason}"
+            )
+
+    return result
+
 
-    result = f"Exiles found for <@{user.id}>:"
+async def get_active_exiles() -> str:
+    logger.info("Starting to run database command")
+    exile_list = exiles_database.get_all_active_exiles()
+    logger.info("db command done")
+    logger.info(type(exile_list))
+    logger.info(len(exile_list))
+    if len(exile_list) == 0:
+        return "No active exiles found"
+
+    result = f"Active exiles found for"
     for exile in exile_list:
-        exile_id = exile[0]
-        exile_reason = exile[1]
-        exile_start_date = exile[2].strftime(TIME_FORMAT)
-        exile_end_date = exile[3].strftime(TIME_FORMAT) if exile[3] else "Indefinite"
-        exile_type = ExileStatus(exile[4]).name
+        logger.info(type(exile))
+        exile_id = exile.exile_id
+        if len(exile.discord_id) == 1:
+            discord_id = f"<@{exile.discord_id[0]}>"
+        else:
+            logger.error("Multiple or non 1 discord ID matched to userID.")
+            discord_id = exile.discord_id
```

Comment:
I think you don't need this if you make the discord_id on the model not a tuple. I don't think there's a reason why there should be more than one discord user ID for an exile, at least with the assumption that we only have 1 server